src/storage/backup/src/lib.rs

```diff
@@ -51,7 +51,7 @@ pub type MetaBackupJobId = u64;
 pub struct MetaSnapshotMetadata {
     pub id: MetaSnapshotId,
     pub hummock_version_id: HummockVersionId,
-    pub ssts: Vec<HummockSstableObjectId>,
```

Comment:
`serde_json` is used to serde `MetaSnapshotMetadata` and then persist it. Change from `Vec` to `HahSet` will break compactibility I think.

Reply:
It's compatible.

Reply:
It's compatible.